src/shared/components/InputBase/InputBase.tsx

```diff
@@ -5,7 +5,7 @@ import { FormGroup, LabelText } from './InputBase.styles'
 export type InputBaseProps = {
   error?: boolean
   warning?: boolean
-  helperText?: string
+  helperText?: string | boolean
```

Comment:
Why would that be a `boolean`?

Reply:
It's was a work around for removing the helper text node that even if empty was adding the container for it and occpuying space, other option cold be to check for empty string arg for not rendering the HelperText node